src/commandDetails/coin/transfer.ts

```diff
@@ -63,12 +63,35 @@ const coinTransferExecuteCommand: SapphireMessageExecuteType = async (
 
   return new SapphireMessageResponseWithMetadata(await transfer.getTransferResponse(), {
     transferId: transfer.transferId,
+    _client: client,
+    receiver: receivingUser,
   });
 };
 
 const transferAfterMessageReply: SapphireAfterReplyType = async (result, sentMessage) => {
   if (typeof result.metadata.transferId === 'undefined') return;
-  // Store the message which the game takes place in the game object
+
+  // Send a dm to the transfer receiver
+  const receivingUser = <User>result.metadata['receiver'];
+  const client = <SapphireClient>result.metadata['_client'];
+
+  const mentionUser = userMention(receivingUser.id);
+
+  // do not attempt sending dms to discord bots
+  if (!receivingUser.bot) {
```

Comment:
If the transfer is to a bot, throw a CodeyUserError since it is an invalid operation